modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -31,6 +45,41 @@ def find_bgs_claim!(claim_id:)
 
           raise
         end
+
+        def find_or_create_ews(bgs_claim)
+          ews = ClaimsApi::EvidenceWaiverSubmission.find_by(claim_id: bgs_claim[:bnft_claim_dto][:bnft_claim_id])
+
+          unless ews&.status&.in?(%w[submitted pending])
```

Comment:
are we worried about the status here? if `ews` is found above, we probably don't care to do anything past that, as we can just return it and update any needed fields. `submitted` also isn't a potential value.

Alongside this, if it does locate an `ews` via the `find_by`, do we check for the status of `updated`? I don't believe BGS would change in any manner, so we could assume that the entire 5103 process has been previously completed if the value were set to `updated`. 
Thinking long term, as a safety net, we should probably at least do the `find` to BGS and let the check above determine if we send the request / pdf, just to make sure. This would be safer in the event that the systems do get out of sync somehow?

Reply:
`unless ews&.status&.in?(%w[updated]) || bgs_claim.nil?` ?

Reply:
the `bgs_claim` shouldn't make it here if it's `nil`, so the latter part is unnecessary.  we should catch it above in
```
          bgs_claim = find_bgs_claim!(claim_id: params[:id])

          if bgs_claim&.dig(:bnft_claim_dto).nil?
            raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found')
          end
```

I'd almost say we do an explicit early return if we find anything in line 50. 

the rest of the method is just the creation, so if it's already created, we just return it at that point.